src/components/UserCards.tsx

```diff
@@ -68,6 +68,21 @@ export default function UserCards({ type, users }: {
   const [searchTerm, setSearchTerm] = useState<string>();
   // Set to null to book with any mentor
   const [bookingMentor, setBookingMentor] = useState<MinUser | null>();
+  const searchInputRef = useRef<HTMLInputElement>(null);
+
+  useEffect(() => {
+    const handleKeyDown = (event: KeyboardEvent) => {
+      if ((event.ctrlKey || event.metaKey) && event.key === 'f') {
```

Comment:
on mac, it should only capture cmd-f and not ctrl-f. vice versa for non-mac oses.